src/components/organisms/ActiveTradeTable/ActiveTradeTable.test.tsx

```diff
@@ -62,7 +62,7 @@ describe('ActiveTradeTable Component', () => {
         expect(sortedRowsDesc[10]).toHaveTextContent('Lend');
     });
 
-    it('should display correct color code for future value', () => {
+    it.skip('should display correct color code for future value', () => {
```

Comment:
I think we can remove this test altogether since future value styling isn't conditional on anything @as472